### PR TITLE
Add output arg to launch files, plus some small fixes

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
- Add output arg to launch files
- Don't set use_sim_time in range_world, but parse it to empty_world
- Add xml prolog to all launch files

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
All checks fail on gazebo_plugins. Which aren't changed in this PR.

Should I also create a PR to merge these commits into lunar-/melodic-devel?

**Comment by MatthijsBurgh** [comment]
ping ;)

**Review by chapulina — DISMISSED** [review]

File: gazebo_ros/launch/range_world.launch [hunk 1]
```diff
@@ -9,17 +9,20 @@
...
     <arg name="world_name" value="$(find gazebo_plugins)/test/test_worlds/$(arg world).world"/>
+    <arg name="paused" value="$(arg paused)"/>
+    <arg name="use_sim_time" value="$(arg use_sim_time)"/>
+    <arg name="extra_gazebo_args" value="$(arg extra_gazebo_args)"/>
     <arg name="gui" value="$(arg gui)"/>
     <arg name="debug" value="$(arg debug)"/>
+    <arg name="physics" value="$(arg physics)"/>
+    <arg name="verbose" value="$(arg verbose)"/>
+    <arg name="output" value="$(arg output)"/>
+    <arg name="world" value="$(arg world)"/>
```
Comment (chapulina) [inline gazebo_ros/launch/range_world.launch]: This is causing test failures:

~~~
14:38:36   /opt/ros/kinetic/share/rostest/cmake/../../../bin/rostest --pkgdir=/tmp/ws/src/gazebo_ros_pkgs/gazebo_plugins --package=gazebo_plugins --results-filename test_range_range_plugin.xml --results-base-dir "/tmp/ws/test_results" /tmp/ws/src/gazebo_ros_pkgs/gazebo_plugins/test/range/range_plugin.test 
14:38:36 ... logging to /home/buildfarm/.ros/log/rostest-8335a4ee6979-3476.log
14:38:36 Traceback (most recent call last):
14:38:36   File "/opt/ros/kinetic/share/rostest/cmake/../../../bin/rostest", line 36, in <module>
14:38:36     rostestmain()
14:38:36   File "/opt/ros/kinetic/lib/python2.7/dist-packages/rostest/__init__.py", line 274, in rostestmain
14:38:36     _main()
14:38:36   File "/opt/ros/kinetic/lib/python2.7/dist-packages/rostest/rostest_main.py", line 159, in rostestmain
14:38:36     testCase = rostest.runner.createUnitTest(pkg, test_file, options.reuse_master, options.clear, options.results_base_dir)
14:38:36   File "/opt/ros/kinetic/lib/python2.7/dist-packages/rostest/runner.py", line 229, in createUnitTest
14:38:36     config = roslaunch.parent.load_config_default([test_file], None)
14:38:36   File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/config.py", line 455, in load_config_default
14:38:36     raise RLException(e)
14:38:36 roslaunch.core.RLException: unused args [world] for include of [/tmp/ws/src/gazebo_ros_pkgs/gazebo_ros/launch/empty_world.launch]
~~~

Comment (MatthijsBurgh) [reply gazebo_ros/launch/range_world.launch]: I removed the argument, lets see if CI succeeds

**Comment by MatthijsBurgh** [comment]
@chapulina CI succeeds, should I also create a PR to merge these commits into lunar-/melodic-devel?

**Comment by chapulina** [comment]
>  should I also create a PR to merge these commits into lunar-/melodic-devel?

Yeah, that would help merging them faster, thanks!

**Review by chapulina — APPROVED** [review]

**Comment by MatthijsBurgh** [comment]
@chapulina see #906 and #907 for Lunar and Melodic

**Pull request merged**